docs/index.adoc

```diff
@@ -218,139 +218,180 @@ libStatic.buildGetter('my/folder', {
+
+.src/main/resources/webapp/webapp.js:
 [source,javascript,options="nowrap"]
 ----
-// src/main/resources/webapp/webapp.js:
-
 const libStatic = require('/lib/enonic/static');
 
 const libRouter = require('/lib/router')();
 
-exports.all = function(req) {
-    return libRouter.dispatch(req);
+exports.all = function(request) {
+    return libRouter.dispatch(request);
 };
 
-// Handling <webappURL>/getResource/...
-libRouter.get( `/getResource/{resourcePath:.+}`,
-    libStatic.buildGetter({
-        root: `'my/folder`,
-        // Override relative path resolution (since request.contextPath is the root of the webapp,
-        // not <webappURL>/getResource/... which is what resource paths should be relative to).
-        // Lib-router provides what we're after - everything after getResource - as
-        // request.pathParams.resourcePath, since we defined that in `/getResource/{resourcePath:.+}`:
-        getCleanPath: request => request.pathParams.resourcePath
+// Set up lib-static:
+const getStatic = libStatic.buildGetter(
+    {
+        root: 'static',
+        getCleanPath: request => request.pathParams.libRouterPath
     }
-));
+);
+libRouter.get(
+    `/static/{libRouterPath:.+}`,
+    request => getStatic(request)
+);
+
 
-// <webappURL> and <webappURL>/
-libRouter.get( `/`, function (request) {
+// Instead of exports.get, libRouter takes over:
+libRouter.get(`/`, request => {
+    const assetsRootUrl = `${request.contextPath}/static`;
     return {
         body: `
-            <html>
-              <head>
-                <title>It still works</title>
-                <link   rel="stylesheet"
-                        type="text/css"
-                        href="${request.contextPath}/getResource/styles.css"
-                />
-              </head>
-
-              <body>
-                  <h1>It still works!</h1>
-                  <img src="${request.contextPath}/getResource/logo.jpg" />
-                  <script src="${request.contextPath}/getResource/js/myscript.js"></script>
-              </body>
-            </html>
-            `
+<html>
+    <head>
+        <title>Webapp</title>
+        <link rel="stylesheet" type="text/css" href="${assetsRootUrl}/styles.css" />
+    </head>
+
+    <body>
+        <h1>My webapp</h1>
+        <img src="${assetsRootUrl}/images/my-logo.jpg" />
+    </body>
+</html>
+        `
     };
 });
 ----
 
-{zwsp} +
-
 [NOTE]
 ====
-It might seem tempting to just let the links in the HTML (`${request.contextPath}/getResource/...`) start with `getResource/`. That looks neater and simpler and could just let the browser append them as relative links, and resolve its requests to `<webappURL>/getResource/...` etc.
+Another gotcha: in this example, `request.contextPath` is used as a simple but stable way to get the webapp's own root URL, aka `<webappRoot>`. Other approaches may be needed for other contexts (e.g. vhosting, etc), but either way:
```

Comment:
There is one more gotcha: Before webapp steps in, XP tries to serve files from `assets` folder. it will be removed in XP 8 https://github.com/enonic/xp/issues/8686

Reply:
Hmmm. Do you think that's within the scope of these docs? I guess it would be in case someone has an asset with the same name (path) as a lib-static route... possible. 

I wonder if anyone uses that feature, and *needs* it... ☢️

Reply:
It is "sort-of" documented, even. My First Webapp uses it.

Reply:
Okay.